src/components/DashboardCard.tsx

```diff
@@ -12,34 +13,45 @@ import { ChainTypes } from '../types/sourceTargetTypes';
 
 interface Props {
     chainType: ChainTypes,
-    useApiContext: Function
+	useApiContext: Function,
+	className? : string
 }
 
 const DashboardCard = ({
 	chainType,
-	useApiContext
+	useApiContext,
+	className
 }: Props) => {
 	const { destination, local } = useDashboardProfile(chainType);
 	const { bestBlockFinalized, bestBlock, importedHeaders: bestBridgedFinalizedBlock,outboundLanes: {  totalMessages, pendingMessages }, inboundLanes: { bridgeReceivedMessages }  } = useDashboard({ destination,local , useApiContext });
 
 	const headerText = chainType === SOURCE ? 'Source' : 'Target';
 	return (
-		<Card style={{ 'wordWrap': 'break-word' }}>
-			<Card.Content header={`${headerText}: ${local}`} />
-			<Card.Description>
-				<div>Best Block: {bestBlock}</div>
-				<div>Best Finalized block: {bestBlockFinalized}</div>
-
-				<div>Pending Messages: {pendingMessages}</div>
-				<div>Total Messages: {totalMessages}</div>
-
-				<hr style={{ 'maxWidth': '80%' }} />
-				<div>Best Target Finalized block: {bestBridgedFinalizedBlock}</div>
-				<div>Received Messages: {bridgeReceivedMessages}</div>
-			</Card.Description>
-			<Card.Content extra>
-			</Card.Content>
-		</Card>);}
+		<Container className={className}>
+			<Card className='container'>
+				<Card.Content header={`${headerText}: ${local}`} />
+				<Card.Description  className='description'>
```

Comment:
```suggestion
				<Card.Description className='description'>
```
Can we use some kind of formatter (also on CI) to make sure this is uniform?

Reply:
yeah, of course ... actually i didn't detected this until now. I suspect that with the machine change i missed some configuration in visual studio code.